.github/workflow_scripts/build_image_prediction_tutorial.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
general comment: `build_xxx.sh` scripts all have same structure. We should refactor-out same customizable functions into a separate function and just source it. Those functions can accept arguments if needed.
This would reduce maintenance costs if a change has to be made to all the tasks.